.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Build fwdump-docker container
+on:
+  push:
+    branches:
+      - '**'        # matches every branch
+  create:
+    tags:
+      - '^[0-9]+\.[0-9]+\.[0-9]+'
+  pull_request:
+    branches:
+      - '**'        # matches every branch
+
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
```

Comment:
We probably do not need QEMU and Buildx. These are useful for cross-compiling containers for different architectures IIRC.